### ISSUE TITLE
Include TEST_DATABASE_URL in docs and .env.example

### ISSUE THREAD

**Issue by Darth-Knoppix** [redwoodjs/redwood#1620]
When using Postgres and running tests with `yarn rw test api`, I receive the following error:

```stdout
error: Error validating datasource `DS`: The URL for datasource `DS` must start with the protocol `postgresql://`.
```

Taking a look at the cli command source, it's expecting a test database url (`TEST_DATABASE_URL`) to be defined if the Prisma provider is Postgres.

If this is intended, it might be a useful to mention this behaviour in the docs and/or add it to _.env.example_ to save some time.

**Comment by jvanbaarsen** [comment]
Added a PR for this: https://github.com/redwoodjs/create-redwood-app/pull/123

**Comment by thedavidprice** [comment]
Will be included in v0.23.0 release
